examples/serdes/dev/system.clj

```diff
@@ -0,0 +1,67 @@
+(ns system
+  "Functions to start and stop the system, used for interactive
+  development.
+
+  The `system/start` and `system/stop` functions are required by the
+  `user` namespace and should not be called directly."
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
```

Comment:
This file is a very good candidate for integrant and integrant.repl

https://github.com/weavejester/integrant/
https://github.com/weavejester/integrant-repl

specially if we leave it as dev only dependency

Reply:
Yes but its not needed for an example IMO. I am generally dependency adverse :P

Reply:
I'd be fine with integrant if it DRYs out the code but not for this PR.